src/views/Pools/components/PoolsTable/ActionPanel/ActionPanel.tsx

```diff
@@ -0,0 +1,241 @@
+  display: flex;
+  flex-direction: column;
+
+  ${({ theme }) => theme.mediaQueries.sm} {
+    flex-direction: row;
+    align-items: center;
+    flex-grow: 1;
+    flex-basis: 0;
+  }
+`
+
+type MediaBreakpoints = {
+  isXs: boolean
+  isSm: boolean
+  isMd: boolean
+  isLg: boolean
+  isXl: boolean
+}
+
+interface ActionPanelProps {
+  account: string
+  pool: Pool
+  isAutoVault: boolean
+  userDataLoaded: boolean
+  expanded: boolean
+  breakpoints: MediaBreakpoints
+}
+
+const InfoSection = styled(Box)`
+  flex: 0 0 210px;
+  padding: 8px 8px;
+  ${({ theme }) => theme.mediaQueries.lg} {
+    padding: 0;
+  }
+`
+
+const ActionPanel: React.FC<ActionPanelProps> = ({
+  account,
+  pool,
+  isAutoVault,
+  userDataLoaded,
+  expanded,
+  breakpoints,
+}) => {
+  const { sousId, stakingToken, earningToken, startBlock, totalStaked, endBlock, isFinished } = pool
+  const { t } = useTranslation()
+  const { currentBlock } = useBlock()
+  const { isXs, isSm, isMd } = breakpoints
+
+  const shouldShowBlockCountdown = Boolean(!isFinished && startBlock && endBlock)
+  const blocksUntilStart = Math.max(startBlock - currentBlock, 0)
+  const blocksRemaining = Math.max(endBlock - currentBlock, 0)
+  const hasPoolStarted = blocksUntilStart === 0 && blocksRemaining > 0
+  const blocksToDisplay = hasPoolStarted ? blocksRemaining : blocksUntilStart
+
+  const isMetaMaskInScope = !!(window as WindowChain).ethereum?.isMetaMask
+  const tokenAddress = earningToken.address ? getAddress(earningToken.address) : ''
+  const imageSrc = `${BASE_URL}/images/tokens/${earningToken.symbol.toLowerCase()}.png`
+
+  const {
+    totalCakeInVault,
+    fees: { performanceFee },
+  } = useCakeVault()
+
+  const performanceFeeAsDecimal = performanceFee && performanceFee / 100
+  const isManualCakePool = sousId === 0
+
+  const getTotalStakedBalance = () => {
+    if (isAutoVault) {
+      return getBalanceNumber(totalCakeInVault, stakingToken.decimals)
+    }
+    if (isManualCakePool) {
+      const manualCakeTotalMinusAutoVault = new BigNumber(totalStaked).minus(totalCakeInVault)
+      return getBalanceNumber(manualCakeTotalMinusAutoVault, stakingToken.decimals)
+    }
+    return getBalanceNumber(totalStaked, stakingToken.decimals)
+  }
+
+  const { targetRef, tooltip, tooltipVisible } = useTooltip(
+    t('Total amount of %symbol% staked in this pool', { symbol: stakingToken.symbol }),
+    {
+      placement: 'bottom',
+    },
+  )
+
+  const blocksRow =
+    blocksRemaining || blocksUntilStart ? (
+      <Flex mb="8px" justifyContent="space-between">
+        <Text mr="4px">{t('Ends in')}:</Text>
+        <Flex>
+          <Link external href={`${BASE_BSC_SCAN_URL}/block/countdown/${endBlock}`}>
+            <Balance fontSize="16px" value={blocksToDisplay} decimals={0} color="primary" />
+            <Text ml="4px" color="primary">
+              {t('blocks')}
+            </Text>
+            <TimerIcon ml="4px" color="primary" />
```

Comment:
Some visual feedback on hover for this icon might be nice to indicate that it's clickable? I'm not sure the Block Balance & text is actually triggering the link? The timer icon is though!